dbt/include/databricks/macros/adapters/catalog.sql

```diff
@@ -69,23 +69,23 @@
 {%- endmacro %}
 
 {% macro databricks__get_catalog_schemas_where_clause_sql(catalog, schemas) -%}
-    where table_catalog = '{{ catalog }}' and ({%- for relation in schemas -%}
-        table_schema = lower('{{ relation[1] }}'){%- if not loop.last %} or {% endif -%}
+    where table_catalog = '{{ catalog|lower }}' and ({%- for relation in schemas -%}
+        table_schema = '{{ relation[1]|lower }}'{%- if not loop.last %} or {% endif -%}
     {%- endfor -%})
 {%- endmacro %}
 
 
 {% macro databricks__get_catalog_relations_where_clause_sql(catalog, relations) -%}
-    where table_catalog = '{{ catalog }}' and (
+    where table_catalog = '{{ catalog|lower }}' and (
         {%- for relation in relations -%}
             {% if relation.schema and relation.identifier %}
                 (
-                    table_schema = lower('{{ relation.schema }}')
-                    and table_name = lower('{{ relation.identifier }}')
+                    table_schema = '{{ relation.schema|lower }}'
+                    and table_name = '{{ relation.identifier|lower }}'
```

Comment:
what if the table name is configured as case-sensitive

Reply:
Can you link to ref?  This would be a considerable monkey wrench if so.

Reply:
nvm. Just found Databricks table name is always case insenstive.